### PR TITLE
Updated src/Controller files for PEP-8 compliance.

### PR DESCRIPTION
**Pull request by JThomas2020**
This PR updates the format of all files in src/Controller to be compliant with PEP-8. This renaming variables and wrapping lines to be complaint. No changes to code functionality, this is intended to increase readibility.

### CODE REVIEW

**Review by Tamookk — APPROVED** [review]
Encountered no problems during testing.

**Comment by didymo** [comment]
conflicts need resolution

**Comment by JThomas2020** [comment]
@didymo conflicts have been resolved

File: src/Controller/ActionHandler.py [hunk 1]
```diff
@@ -168,45 +183,55 @@ def init_windowing_menu(self):
...
         """
         Function triggered when a window is selected from the menu.
-        :param state: Variable not used. Present to be able to use a lambda function.
+        :param state: Variable not used. Present to be able to use a lambda
+        function.
         :param text: The name of the window selected.
         """
         # Get the values for window and level from the dict
-        windowing_limits = self.patient_dict_container.get("dict_windowing")[text]
+        windowing_limits = self.patient_dict_container.get("dict_windowing")[
```
Comment (tdlinh2712) [inline src/Controller/ActionHandler.py:226]: i think breaking the line there doesn't make sense. how about breaking the line at `.get("dict_windowing")[text]`?

Comment (JThomas2020) [reply src/Controller/ActionHandler.py:226]: Thats a good idea, PEP-8 prefers splitting the line at the bracket, but '\' notation is acceptable so I'll do that

**Review by tdlinh2712 — APPROVED** [review]

**Pull request merged**